lib/cartopy/crs.py

```diff
@@ -1531,6 +1531,85 @@ def y_limits(self):
         return self._ylim
 
 
+class AlbersEqualArea(Projection):
+    """
+    An Albers Equal Area projection
+
+    This projection is conic and equal-area, and is commonly used for maps of
+    the conterminous United States.
+    """
+
+    def __init__(self, central_longitude=0.0, central_latitude=0.0,
+                 false_easting=0.0, false_northing=0.0,
+                 standard_parallels=(20.0, 50.0), globe=None):
+        """
+        Kwargs:
+
+            * central_longitude - The central longitude. Defaults to 0.
+            * central_latitude - The central latitude. Defaults to 0.
+            * false_easting - X offset from planar origin in metres.
+                              Defaults to 0.
+            * false_northing - Y offset from planar origin in metres.
+                               Defaults to 0.
+            * standard_parallels - The one or two latitudes of correct scale.
```

Comment:
I prefer standard parallels.

References: http://support.esri.com/en/knowledgebase/GISDictionary/term/standard%20parallel
And: http://www.geography.hunter.cuny.edu/~jochen/GTECH361/lectures/lecture04/concepts/Map%20coordinate%20systems/Lambert%20Conformal%20Conic.htm (specifically the image:
![LambertConformalConic](http://www.geography.hunter.cuny.edu/~jochen/GTECH361/lectures/lecture04/concepts/Map%20coordinate%20systems/Lambert%20Conformal%20Conic_files/image002.gif)
)